docs/stdlib/sequence.rst

```diff
@@ -66,12 +66,12 @@ Sequences
                   std::sequence_reset( \
                     seq: schema::ScalarType, val: int64) -> int64
 
-    Reset the sequence to its initial state or the specified value.
+    Returns the initial state of a sequence or a specified ``val``.
```

Comment:
```suggestion
    Resets a sequence to initial state or a given value, returning the value.
```

Not in love with this one, but I believe it encapsulates everything and fits within our space constraint.